docs/client-standalone.md

```diff
@@ -19,11 +19,10 @@ This is a full script runnable directly with [JBang](https://www.jbang.dev/) tha
 uses a dynamic client for connecting to [countries.trevorblades.com](https://countries.trevorblades.com)
 to obtain a list of countries from its database.
 
-```
+```java
 ///usr/bin/env jbang "$0" "$@" ; exit $?
-//DEPS io.smallrye:smallrye-graphql-client-implementation-vertx:1.5.0
+//DEPS io.smallrye:smallrye-graphql-client-implementation-vertx:RELEASE
```

Comment:
I don't think that JBang supports this version pointer, I get 
```
[jbang] [ERROR] Could not resolve dependencies: Failed to collect dependencies at io.smallrye:smallrye-graphql-client-implementation-vertx:jar:LATEST
```
when running it

Reply:
I can revert to:

```
//DEPS io.smallrye:smallrye-graphql-client-implementation-vertx:2.10.0
```

if you want, but for this version it will not work without:

```
//DEPS org.eclipse:yasson:2.0.4
```

And it would be better to have always the latest version in the documentation. Somehow with `RELEASE` it is ever green.

Reply:
There are even integration tests in the jbang project using the `RELEASE` notation:

https://github.com/jbangdev/jbang/blob/7af0ad56db2162121ab48b37132008417528a7c0/itests/SankeyPlotTestWithDeps.java#L16-L18

Reply:
Ok somehow it works for me now with RELEASE (no idea why it didn't yesterday), but it picks up 2.10.0, so it still needs the dependency... But hey, I think we will release 2.11 (or 2.10.1) within a few weeks, so I guess we could just keep they way you have it now, and hopefully it will start working by itself then

Reply:
Now that I think of it, the documentation will also only deploy when we do a release, so with the next documentation update, it should just start working